pxd.c

```diff
@@ -2120,6 +2125,7 @@ static int pxd_control_release(struct inode *inode, struct file *file)
 		pxd_printk("%s: not opened\n", __func__);
 	} else {
 		ctx->fc.connected = 0;
+		wmb();
```

Comment:
there is a barrier inside schedule_delayed_work this is redundant

Reply:
wmb even if redundant assures of the modified value.